tests/e2e/warehouse/warehouse_test.go

```diff
@@ -0,0 +1,131 @@
+			Expect(helpers.IsThisInThatSliceString(destinationsIDs, loadedDestinationIDs)).Should(Equal(true))
+		})
+	})
+	Describe("Compatible with segment warehouse schema", func() {
+		BeforeEach(func() {
+			helpers.DeleteRowsInTable(dbHandle, warehouseLoadFilesTable)
+		})
+		It("should be able to create tables ", func() {
+			helpers.SendTrackRequest("1YyeVOLDReXPNYORjDlvE7PM1Jw", helpers.AddKeyToJSON(helpers.RemoveKeyFromJSON(helpers.TrackPayload, "messageId", "anonymousId"), "event", eventName))
+			helpers.SendIdentifyRequest("1YyeVOLDReXPNYORjDlvE7PM1Jw", helpers.RemoveKeyFromJSON(helpers.IdentifyPayload, "messageId", "anonymousId"))
+			helpers.SendPageRequest("1YyeVOLDReXPNYORjDlvE7PM1Jw", helpers.RemoveKeyFromJSON(helpers.PagePayload, "messageId", "anonymousId"))
+			//helpers.SendAliasRequest("1YyeVOLDReXPNYORjDlvE7PM1Jw", helpers.RemoveKeyFromJSON(helpers.AliasPayload, "messageId", "anonymousId"))
+			//helpers.SendGroupRequest("1YyeVOLDReXPNYORjDlvE7PM1Jw", helpers.RemoveKeyFromJSON(helpers.GroupPayload, "messageId", "anonymousId"))
+			helpers.SendScreenRequest("1YyeVOLDReXPNYORjDlvE7PM1Jw", helpers.RemoveKeyFromJSON(helpers.ScreenPayload, "messageId", "anonymousId"))
+			tables := []string{"identifies", "users", "pages", "tracks", "screens", strings.Replace(strings.ToLower(eventName), " ", "_", -1)} // group, alias are not supported.
+			Eventually(func() bool {
+				loadedTables := helpers.GetLoadFileTableName(dbHandle, warehouseLoadFilesTable)
+				return helpers.IsThisInThatSliceString(tables, loadedTables)
+			}, loadTablesTimeout, pollIntervalForLoadTables).Should(Equal(true))
+		})
+	})
+	Describe("testing with different string formats", func() {
+		BeforeEach(func() {
+			helpers.DeleteRowsInTable(dbHandle, warehouseLoadFilesTable)
+		})
+		It("should be able to create load file", func() {
+			helpers.SendBatchRequest("1YyeVOLDReXPNYORjDlvE7PM1Jw", helpers.AddKeyToJSON(helpers.RemoveKeyFromJSON(helpers.DiffStringFormatBatchPayload, "batch.0.messageId", "batch.0.anonymousId"), "batch.0.event", eventName))
+			eventName := strings.Replace(strings.ToLower(eventName), " ", "_", -1)
+			Eventually(func() bool {
+				loadedTables := helpers.GetLoadFileTableName(dbHandle, warehouseLoadFilesTable)
+				return helpers.IsThisInThatSliceString([]string{eventName}, loadedTables)
+			}, loadTablesTimeout, pollIntervalForLoadTables).Should(Equal(true))
+		})
+	})
+	Describe("sending different data types for a key consecutively", func() {
+		BeforeEach(func() {
+			helpers.DeleteRowsInTable(dbHandle, warehouseLoadFilesTable)
+			helpers.DeleteRowsInTable(dbHandle, warehouseSchemasTable)
+		})
+		It("should be able to create load file and schema should have different data types", func() {
+			helpers.SendTrackRequest("1YyeVOLDReXPNYORjDlvE7PM1Jw", helpers.AddKeyToJSON(helpers.RemoveKeyFromJSON(gjson.Get(helpers.DTBatchPayload, "batch.0").String(), "messageId", "anonymousId"), "event", eventName))
```

Comment:
1. test if load is successful in wh_uploads after data type change for a column
2. check load file json to verify that new value is set to null on data type chamge